test_scour.py

```diff
@@ -1502,7 +1502,16 @@ def runTest(self):
         doc = scourXmlFile('unittests/redundant-svg-namespace.svg').documentElement
         r = doc.getElementsByTagNameNS(SVGNS, 'rect')[1]
         self.assertEqual(r.tagName, 'rect',
-                         'Redundant svg: prefix not removed')
+                         'Redundant svg: prefix not removed from rect')
+        t = doc.getElementsByTagNameNS(SVGNS, 'text')[0]
+        self.assertEqual(t.tagName, 'text',
+                         'Redundant svg: prefix not removed from text')
+
+        # Regression test for #239
+        self.assertEqual(t.getAttribute('xml:space'), 'preserve',
+                         'Redundant svg: prefix removed from xml:space')
```

Comment:
Could you change this and the next comment to describe the issue better? ("required xml: prefix removed in error" or similar)